src/components/BigNumberPad.js

```diff
@@ -0,0 +1,72 @@
+import React, {PureComponent} from 'react';
+import {
+    Text, TouchableOpacity, View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+
+const propTypes = {
+    numberPressed: PropTypes.func.isRequired,
+};
+
+const padNumbers = [
+    ['1', '2', '3'],
+    ['4', '5', '6'],
+    ['7', '8', '9'],
+    ['.', '0', '<'],
```

Comment:
~@Julesssss why do we have `<` ? 😕~

Oh I see, it's backspace... Could we instead maybe use something like `←` ?

Reply:
You'll have to present a good case to @shawnborton 😄

Reply:
Should I change it to another character or leave it as is?

Reply:
Deferring to @shawnborton

Reply:
I believe Shawn is OOO for some of this week. Let's continue as-is for now as we can make this change at any time.